csp_tools/aws/Dockerfile

```diff
@@ -12,10 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvcr.io/ea-bignlp/ga-participants/nemofw-training:23.08.03
+FROM nvcr.io/ea-bignlp/nemofw-training:23.07-py3
 
-ARG EFA_INSTALLER_VERSION=latest
-ARG AWS_OFI_NCCL_VERSION=1.4.0-aws
+ARG NCCL_VERSION=2.18.5-1+cuda12.2
+ARG EFA_INSTALLER_VERSION=1.24.1
+ARG AWS_OFI_NCCL_VERSION=1.7.2-aws
+
+# NCCL
+RUN cd /opt && \
+    wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-keyring_1.0-1_all.deb && \
```

Comment:
is `cuda-keyring` not installed in the base container?

Reply:
the most updated version is not installed. This is the only way to get the 2.18.5+cuda12.2 libnccl packages.